thesis-api/main/services/question_service/question_service.py

```diff
@@ -0,0 +1,44 @@
+import random
+
+from fastapi import Depends
+
+from persistence.objects import Question
+from persistence.word_dao import WordDAO
+from services.question_service.definition_processing_service import DefinitionProcessingService
+from services.question_service.definition_service import WikipediaDefinitionService
+from services.question_service.wrong_answers_service import DatamuseWrongAnswerService
+
+
+class QuestionService:
+    def __init__(self, dao: WordDAO = Depends(), definition_service: WikipediaDefinitionService = Depends()):
```

Comment:
I would suggest adding `@staticmethod build()` like in services on master branch.